### ISSUE TITLE
Mapbox GL JS

### ISSUE THREAD

**Issue by azran4u** [developmentseed/dirty-reprojectors#14]
Hi,

I'm using mapbox-gl-js with tileset projected EPSG:4326.
My dataset is geoJSON WGS84.
Since Mapbox tries to display data as web mercator, data is not positioned in the correct location on map. 
I'm trying to use dirty-reproject to convert the input geoJSON to output geoJSON so Mapbox displays it correctly. Yet, the command dirty-reproject --list doesn't have an epsg:4326 option.
Thanks,
Eyal

**Comment by dereklieu** [comment]
@azran4u I always get my wgs'es mixed up with my epsg'es, but I believe your issue may be something in how your geojson data is processed. There is no difference between WGS84 and EPSG4326 that would be relevant to this module, so it's not included as an option.

https://gis.stackexchange.com/questions/3334/difference-between-wgs84-and-epsg4326

**Comment by azran4u** [comment]
I agree with you, yet i'm trying  to reproject a geojson so that mapbox gl will render it looking like 'epsg:4326'.

**Comment by azran4u** [comment]
by looking like 'epsg:4326' I mean Plate-Carrée

**Comment by dereklieu** [comment]
I see. Since this module uses D3 geo libraries for projection math, the projections that would be straightforward to achieve are all listed under the [d3-geo-projection](https://github.com/d3/d3-geo-projection) and [d3-geo](https://github.com/d3/d3-geo) libraries. Perhaps these libraries have been updated since we developed this module, and more projections are available. If one of the ones listed at either of the links serves your purposes, but isn't listed as an option, I can look into integrating it.
